dap_prinz_green_jobs/tests/test_industry_measures.py

```diff
@@ -49,56 +75,23 @@ def test_clean_unit_emissions_dict():
 
 def test_industry_measures():
     im = IndustryMeasures()
+    im.load()
+
+    industry_measures = im.get_measures(job_ads)
+
+    assert len(industry_measures) == len(job_ads)
+    assert industry_measures[0]["SIC"] == "62090"
+    assert industry_measures[1]["SIC"] == "32130"
+    assert industry_measures[3]["SIC"] == None
 
-    # Just use a very small replicated part of the data so we can see it's working as we'd like
-    im.ojo_companies_house_dict = {
-        "boots": [
-            {
-                "CompanyName": "BOOTS INTERNATIONAL LIMITED",
-                "SICCode.SicText_1": "46450 - Wholesale of perfume and cosmetics",
-            }
-        ],
-        "j sainsbury": [
-            {
-                "CompanyName": "J SAINSBURY PLC",
-                "SICCode.SicText_1": "47110 - Retail sale in non-specialised stores with food, beverages or tobacco predominating",
-            }
-        ],
-        "global british petroleum": [
-            {
-                "CompanyName": "GLOBAL BRITISH PETROLEUM LTD",
-                "SICCode.SicText_1": "09100 - Support activities for petroleum and natural gas extraction",
-                "SICCode.SicText_2": "46120 - Agents involved in the sale of fuels, ores, metals and industrial chemicals",
-                "SICCode.SicText_3": "46690 - Wholesale of other machinery and equipment",
-                "SICCode.SicText_4": "46750 - Wholesale of chemical products",
-            }
-        ],
-    }
-
-    im.ghg_emissions_dict = {"464": 6299.6, "09": 28.2}
-    im.sic_section_2_prop_hours = {"G": 9.4}
-
-    assert im.get_green_measure_for_company("Boots")["SIC"] == "46450"
-    assert (
-        im.get_green_measure_for_company("Boots")["INDUSTRY TOTAL GHG EMISSIONS"]
-        == 6299.6
-    )
-    assert (
-        im.get_green_measure_for_company("Boots")["INDUSTRY PROP HOURS GREEN TASKS"]
-        == 9.4
-    )
-    assert (
-        im.get_green_measure_for_company("Global British Petroleum")["SIC"] == "09100"
-    )
-    assert (
-        im.get_green_measure_for_company("Global British Petroleum")[
-            "INDUSTRY TOTAL GHG EMISSIONS"
-        ]
-        == 28.2
-    )
     assert (
-        im.get_green_measure_for_company("Global British Petroleum")[
-            "INDUSTRY PROP HOURS GREEN TASKS"
-        ]
-        == None
+        industry_measures[2]["SIC_name"]
+        == "Sale of motor vehicle parts and accessories"
     )
+    assert industry_measures[2]["INDUSTRY PROP HOURS GREEN TASKS"] == None
```

Comment:
with the fix to `sic_to_section`
```suggestion
    assert industry_measures[2]["INDUSTRY PROP HOURS GREEN TASKS"] == 9.4
```